utils/rpms/daos.spec

```
@@ -346,6 +349,7 @@ getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r -g daos_serv
 %post server
 /sbin/ldconfig
 %systemd_post %{server_svc_name}
+sysctl -p %{_sysconfdir}/sysctl.d/%{sysctl_script_name}
```

There is a [macro specifically for this](https://docs.fedoraproject.org/en-US/packaging-guidelines/#_binfmt_d_sysctl_d_and_tmpfiles_d):

```suggestion
%sysctl_apply %{sysctl_script_name}
```